src/interfaces/restoration-interfaces/restoration-interface.ts

```diff
@@ -16,6 +16,7 @@ export interface RestorationIF {
   nameTranslations?: NameTranslationIF[]
   offices: AddressesIF
   parties: OrgPersonIF[]
-  relationships?: string[]
+  noticeDate?: string // YYYY-MM-DD
+  relationships?: RelationshipTypes[]
```

Comment:
While you're here, please remove the ? from the required properties (and add the missing prop?) << @jonathan-longe Is `type` your prop?

Ref: https://github.com/bcgov/business-schemas/blob/1f068c23df4427d2ada72dc16c869884b105d24c/src/registry_schemas/schemas/restoration.json#L13

Reply:
Somehow missed this. Fixing it now. @jonathan-longe, could you please let me know if `type` is your prop?

Reply:
Nope, I don't recognize it.  It looks like `type` was added by @JazzarKarim

Reply:
Didn't add it also! It was already there when I worked on my PR. What's the issue exactly?

Reply:
I was asking about required vs optional props, but there's another conversation for this below so I'm going to close this conversation.